src/components/EmptyView/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './EmptyViewStyle';
+import { Button } from 'react-native-paper';
+import { connect } from 'react-redux';
+import { RootState } from '../../redux/reducers';
+import { Props } from './types';
+
+function EmptyView(props: Props) {
+  const {
+    fullScreenLoadingIndicator,
+    title,
+    description,
+    uri,
+    onPress,
+    isRefresh
+  } = props;
+  return !fullScreenLoadingIndicator?.visible ? (
+    <View style={styles.container}>
+      <Image
+        source={uri ? uri : require('../../assets/images/packing.png')}
+        style={styles.stashEmpty}
+        resizeMode={'contain'}
+      />
+      <Text style={styles.titleText}>{title}</Text>
```

Comment:
Should not there be a default title and default description?

Reply:
@awalkowiak No there is no default title and descrption value